src/temp/Project/SearchTabForMobile.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import StarIcon from '../icons/StarIcon';
+import SearchIcon from '../icons/SearchIcon';
+import FilterIcon from '../icons/FilterIcon';
+import ListIcon from '../icons/ListIcon';
+import LocationIcon from '../icons/LocationIcon';
+import style from '../Project/Search.module.scss';
+import { useTranslation } from 'next-i18next';
+
+interface SearchTabForMobileProps {
+  active: boolean;
+}
+
+export const SearchTabForMobile = ({ active }: SearchTabForMobileProps) => {
+  const { t } = useTranslation(['projectDetails']);
+  const [isAllProject, setIsAllProject] = useState(false);
+  const [isTopProject, setIsTopProject] = useState(true);
```

Comment:
You could probably combine the state here

`const [tabSelected, setTabSelected] = useState<'topProjects' | 'allProjects'>('topProjects');`